packages/account/src/Components/poi/status/unsupported/detail-component.tsx

```diff
@@ -118,7 +125,14 @@ const DetailComponent = ({
         case STATUS.IS_UPLOADING:
             return <Loading is_fullscreen={false} is_slow_loading status={[localize('Uploading documents')]} />;
         case STATUS.IS_COMPLETED:
-            return <UploadComplete is_from_external={true} needs_poa={false} is_manual_upload />;
+            return (
+                <VerificationStatus
+                    icon={manual_upload_complete_status_content.icon}
+                    is_mobile={isMobile()}
```

Comment:
can we move this to is_mobile?

Reply:
this component is not using useStore,
Let’s do it in refactoring card related to manual flow